[BUG] Creation of Service Binding for Cloud Logging Service crashes

### Is there an existing issue for this?

- [X] I have searched the existing issues

### What version of the Terraform provider are you using?

1.1.0

### What version of the Terraform CLI are you using?

1.7.2

### What type of issue are you facing

bug report

### Describe the bug

The creation of a service binding for the service `cloud-logging` fails, namely crashes after it triggered the creation of the binding.
The request for the creation of the binding is sent to the backend (and the binding gets created) However, the loop in the provider waiting for successful creation fails i,e, crashes:

```bash
Stack trace from the terraform-provider-btp_v1.1.0 plugin:

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0xb7c7b0]

goroutine 355 [running]:
github.com/SAP/terraform-provider-btp/internal/provider.subaccountServiceBindingValueFrom({_, _}, {{0xc0004b64b0, 0x24}, 0x0, 0x0, {0x0, 0x0}, {0x0, 0x0}, ...})
        github.com/SAP/terraform-provider-btp/internal/provider/type_subaccount_service_binding.go:38 +0x130
github.com/SAP/terraform-provider-btp/internal/provider.(*subaccountServiceBindingResource).Create(0xc000184408, {0xea0418, 0xc0002e2fc0}, {{{{0xea53d8, 0xc00026ac30}, {0xc33c20, 0xc00026aa50}}, {0xea8378, 0xc00011c550}}, {{{0xea53d8, ...}, ...}, ...}, ...}, ...)
        github.com/SAP/terraform-provider-btp/internal/provider/resource_subaccount_service_binding.go:193 +0x711
github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).CreateResource(0xc0000f2d00, {0xea0418, 0xc0002e2fc0}, 0xc0004f35f0, 0xc0004f35c8)
        github.com/hashicorp/terraform-plugin-framework@v1.6.0/internal/fwserver/server_createresource.go:101 +0x578
github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).ApplyResourceChange(0xc0000f2d00, {0xea0418, 0xc0002e2fc0}, 0xc00011d5e0, 0xc0004f36e0)
        github.com/hashicorp/terraform-plugin-framework@v1.6.0/internal/fwserver/server_applyresourcechange.go:57 +0x4aa
github.com/hashicorp/terraform-plugin-framework/internal/proto6server.(*Server).ApplyResourceChange(0xc0000f2d00, {0xea0418?, 0xc0002e2ed0?}, 0xc00011d540)
        github.com/hashicorp/terraform-plugin-framework@v1.6.0/internal/proto6server/server_applyresourcechange.go:55 +0x38e
github.com/hashicorp/terraform-plugin-go/tfprotov6/tf6server.(*server).ApplyResourceChange(0xc0000e86e0, {0xea0418?, 0xc0002e2420?}, 0xc00020c540)
        github.com/hashicorp/terraform-plugin-go@v0.22.0/tfprotov6/tf6server/server.go:846 +0x3d0
github.com/hashicorp/terraform-plugin-go/tfprotov6/internal/tfplugin6._Provider_ApplyResourceChange_Handler({0xd189e0, 0xc0000e86e0}, {0xea0418, 0xc0002e2420}, 0xc00015d380, 0x0)
        github.com/hashicorp/terraform-plugin-go@v0.22.0/tfprotov6/internal/tfplugin6/tfplugin6_grpc.pb.go:518 +0x1a6
google.golang.org/grpc.(*Server).processUnaryRPC(0xc000374000, {0xea0418, 0xc0002e2390}, {0xea6b80, 0xc0000f3040}, 0xc0005f7c20, 0xc0002e3bf0, 0x13ed4b8, 0x0)
        google.golang.org/grpc@v1.61.1/server.go:1385 +0xdd1
google.golang.org/grpc.(*Server).handleStream(0xc000374000, {0xea6b80, 0xc0000f3040}, 0xc0005f7c20)
        google.golang.org/grpc@v1.61.1/server.go:1796 +0xfb8
google.golang.org/grpc.(*Server).serveStreams.func2.1()
        google.golang.org/grpc@v1.61.1/server.go:1029 +0x8b
created by google.golang.org/grpc.(*Server).serveStreams.func2 in goroutine 12
        google.golang.org/grpc@v1.61.1/server.go:1040 +0x125
``` 

### Expected Behavior

The service binding is successfully created and the state is stored in the tfstate file

### Steps To Reproduce

Here is a Terraform script that does a basic setup of the scenario:

**provider.tf**

```terraform
terraform {

  required_providers {
    btp = {
      source  = "SAP/btp"
      version = "~>1.1.0"
    }
  }
}

provider "btp" {
  globalaccount = "<SUBDOMAIN>"
}

```

**variables.tf**

```terraform
variable "subaccount_name" {
  description = "The name of the subaccount where the service instance should be created"
  type = string
  default = "retest-cloud-logging"
}

variable "region" {
  description = "The region where the service instance should be created"
  type = string
  default = "us10"
}

variable "subdomain" {
  description = "The subdomain of the subaccount where the service instance should be created"
  type = string
  default = "retest-cloud-logging"
}
```

**main.tf**

```terraform
resource "btp_subaccount" "sa_retest_cloud_logging" {
  name      = var.subaccount_name
  region    = var.region
  subdomain = var.subdomain
}

resource "btp_subaccount_entitlement" "se_retest_cloud_logging" {
  subaccount_id = btp_subaccount.sa_retest_cloud_logging.id
  service_name  = "cloud-logging"
  plan_name     = "standard"
}


data "btp_subaccount_service_plan" "sp_cloudlogging_standard" {
  subaccount_id = btp_subaccount.sa_retest_cloud_logging.id
  name          = "standard"
  offering_name = "cloud-logging"
  depends_on = [ btp_subaccount_entitlement.se_retest_cloud_logging ]
}

resource "btp_subaccount_service_instance" "si_cloud_logging_standard" {
  subaccount_id  = btp_subaccount.sa_retest_cloud_logging
  serviceplan_id = data.btp_subaccount_service_plan.sp_cloudlogging_standard.id
  name           = "my-cloud_logging_instance"
  parameters = jsonencode({
    retention_period = 14
    backend = {
      max_data_nodes = 10
      api_enabled    = false
    },
    ingest = {
      max_instances = 10
    }
  })
}

resource "btp_subaccount_service_binding" "sb_cloud_loggign_standard" {
  subaccount_id       = btp_subaccount.sa_retest_cloud_logging.id
  service_instance_id = btp_subaccount_service_instance.si_cloud_logging_standard.id
  name                = "sk1_cloud_logging"
}
```


### User's Role Collections

- Global Account Admin
- Subaccount Admin

### Add screenshots to help explain your problem

n/a

### Additional context

- Region was us10 in production landscape
- Creation of Cloud Logging takes approx. 6 minutes

--- a/internal/provider/type_subaccount_service_binding.go
+++ b/internal/provider/type_subaccount_service_binding.go
@@ -35,9 +35,14 @@ func subaccountServiceBindingValueFrom(ctx context.Context, value servicemanager
 		ServiceInstanceId: types.StringValue(value.ServiceInstanceId),
 		Context:           types.StringValue(string(value.Context)),
 		Credentials:       types.StringValue(string(value.Credentials)),
-		State:             types.StringValue(value.LastOperation.State),
-		CreatedDate:       timeToValue(value.CreatedAt),
-		LastModified:      timeToValue(value.UpdatedAt),
+		//		State:             types.StringValue(value.LastOperation.State),
+		CreatedDate:  timeToValue(value.CreatedAt),
+		LastModified: timeToValue(value.UpdatedAt),
+	}
+
+	// Temporal workaround and explicit mapping to make state handling more resillient
+	if value.LastOperation != nil {
+		serviceBinding.State = types.StringValue(value.LastOperation.State)
 	}
 
 	var diags, diagnostics diag.Diagnostics

--- a/internal/provider/type_subaccount_service_binding.go
+++ b/internal/provider/type_subaccount_service_binding.go
@@ -35,12 +35,11 @@ func subaccountServiceBindingValueFrom(ctx context.Context, value servicemanager
 		ServiceInstanceId: types.StringValue(value.ServiceInstanceId),
 		Context:           types.StringValue(string(value.Context)),
 		Credentials:       types.StringValue(string(value.Credentials)),
-		//		State:             types.StringValue(value.LastOperation.State),
-		CreatedDate:  timeToValue(value.CreatedAt),
-		LastModified: timeToValue(value.UpdatedAt),
+		CreatedDate:       timeToValue(value.CreatedAt),
+		LastModified:      timeToValue(value.UpdatedAt),
 	}
 
-	// Temporal workaround and explicit mapping to make state handling more resillient
+	// CREATE and GET repsonses might differ - safeguarding nil reference of LastOperation
 	if value.LastOperation != nil {
 		serviceBinding.State = types.StringValue(value.LastOperation.State)
 	}

